### ISSUE TITLE
Make all MarkerType not require the stroke color to be set

### ISSUE THREAD

**Issue by maettu-this** [oxyplot/oxyplot#1466]
### Feature description

Why does some markers requires the stroke color to be set? Cannot `OxyColors.Automatic` be good enough? Why does `MarkerFill` explicitly needs to be set to the same color as Color, why cannot it "follow" that color in case only Color is `OxyColors.Automatic`? Making all MarkerType not require the stroke color to be set would ease usage.

**Comment by objorke** [comment]
I agree the default marker color should be consistent everywhere. For those series where markers should not be visible by default (e.g. LineSeries), the `MarkerType` is set to `None`.

The implementations of automatic `MarkerFill` may need to be reviewed, it seems a bit complex: https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/Series/LineSeries.cs#L397

**Comment by objorke** [comment]
I see `MarkerFill` is only used in three series LineSeries, ScatterSeries, TwoColorAreaSeries
